server/routes/nl/html.py

```diff
@@ -43,11 +44,33 @@ def page():
                          website_hash=os.environ.get("WEBSITE_HASH"))
 
 
+def _get_selenium_driver():
+  options = webdriver.chrome.options.Options()
+  options.add_argument("--headless=new")
+  options.add_argument("--disable-gpu")
+  options.add_argument("--no-sandbox")
+  options.add_argument("enable-automation")
+  options.add_argument("--disable-infobars")
+  options.add_argument("--disable-dev-shm-usage")
+  return webdriver.Chrome(options=options)
```

Comment:
how expensive is this? Is there a delay here?

Reply:
Nearly a second.  Might need to explore other options soon.